### DISCUSSION TITLE
s4z update and lost all settings again

### DISCUSSION THREAD

**Discussion by LeebZHCC** [SauceLLC/sauce4zwift#76] [General]
Love S4Z but the last time I updated it lost all my windows and data settings so had to set them all up again.  Yesterday I updated as I could see that we will now have a map and gradient, elevation etc (super chuffed for that) .  Updated and every data field was "elapsed time" so had to reset them all again.

Now knowing I may lose my settings when updating S4Z I downloaded my settings before updating.  After the update and all settings lost, I installed my saved settings but that didn't change anything?  Am I doing something wrong?

**Comment by mayfield** [comment]
Fixed with 0471183095cf3c7b2f96d1c6bade58a0280859a9
